addons/medical_feedback/XEH_postInit.sqf

```diff
@@ -10,6 +10,14 @@
     [_unit, "moan", PAIN_TO_MOAN(_painLevel)] call FUNC(playInjuredSound);
 }] call CBA_fnc_addEventHandler;
 
+[QEGVAR(medical,fracture), {
+    params ["_unit"];
+
+    if (_unit == ACE_player) then {
```

Comment:
Should we make it so that nearby players can also hear a fracture (positional sound)?

Additionally, this means unconscious players will hear a fracture. We may want to make that not the case.